NET_project/Benchmarks/FirefliesFlocking.cs

```diff
@@ -16,13 +16,13 @@ public unsafe struct FirefliesFlockingNET : IJob {
 		result = FirefliesFlocking(boids, lifetime);
 	}
 
-	private uint parkMiller;
-	private float maxSpeed;
-	private float maxForce;
-	private float separationDistance;
-	private float neighbourDistance;
+	private static uint parkMiller;
+	private static float maxSpeed;
+	private static float maxForce;
+	private static float separationDistance;
+	private static float neighbourDistance;
```

Comment:
Notably these are effectively `const` as far as the benchmark uses them. It looks like C/C++ do whole program analysis and can see that they are never mutated elsewhere.

.NET can't do really do that since code is jitted as needed and so it would need the user to mark it `const` or `static readonly` to take advantage.